clustering.Rmd

```diff
@@ -616,18 +745,25 @@ elbow_plot
 ```
 
 It looks like 3 clusters is the right choice for this data.
-But why is there a "bump" in the total WSSD plot here? Shouldn't total WSSD always
-decrease as we add more clusters? Technically yes, but remember:  K-means can 
-get "stuck" in a bad solution. Unfortunately, for K = 8 we had an unlucky initialization
-and found a bad clustering! \index{K-means!restart,nstart} We can help prevent finding a bad clustering by trying a 
-few different random initializations via the `nstart` argument (here we use 10 restarts). 
+But why is there a "bump" in the total WSSD plot here? 
+Shouldn't total WSSD always decrease as we add more clusters? 
+Technically yes, but remember:  K-means can get "stuck" in a bad solution. 
+Unfortunately, for K = 8 we had an unlucky initialization
+and found a bad clustering! \index{K-means!restart,nstart} 
+We can help prevent finding a bad clustering 
+by trying a few different random initializations 
+via the `nstart` argument (here we use 10 restarts). 
+When we do this, K-means clustering will be performed 
+the number of times specified by the `nstart` argument,
+and R will return to us the best clustering from this.
+The more times we perform K-means clustering, 
+the more likely we are to find a lucky, or good clustering (if one exists!).
```

Comment:
replace with: "the more likely we are to find a good clustering (if one exists)."

Reply:
done